src/main/java/org/opentripplanner/model/plan/Itinerary.java

```diff
@@ -51,6 +51,8 @@ public class Itinerary {
   /* Sandbox experimental properties */
   private Float accessibilityScore;
 
+  private Double emissions;
```

Comment:
This need to be wrapped in an `Emissions` class with field `co2 : Double` or `co2 : Gram`. The Gram class does not exist, but you may create on in `org.opentripplanner.framework.model.Gram`. The `Emissions`class should be in the `o.o.model.plan` package and be tagged with `@Sandbox`. The field `co2` should be optional since we probably want to add other emissions later. There is also a difference between no emissions calculated and no co2 emissions calculated.